apps/examples/testcase/le_tc/kernel/tc_task.c

```diff
@@ -520,6 +625,30 @@ static void tc_task_task_init(void)
 	TC_SUCCESS_RESULT();
 }
 
+/**
+* @fn                   :tc_task_task_monitor_utils
+* @brief                :Call task monitor uilities (TCB)
+* @Scenario             :Call task monitor uilities (TCB)
+* API's covered         :task_monitor_register, task_monitor_update_status
+* Preconditions         :none
+* Postconditions        :none
+* @return               :void
+*/
+#ifdef CONFIG_TASK_MONITOR
+static void tc_task_task_monitor_utils(void)
+{
+	int ret_chk;
+	g_callback = ERROR;
+
+	ret_chk = task_monitor_register(SEC_2);
```

Comment:
We cannot use task monitor API from user space.
I think, because of this, you made the tc in prctl.
Please remove this function.